chsdi/templates/htmlpopup/phosphor.mako

```diff
@@ -0,0 +1,5 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+   <tr><td width='200'>${_('ch.bafu.gewaesserschutz-diffuse_eintraege_phosphor.phosphor')}</td><td>${c['properties']['phosphor'] or '-'}</td></tr>
```

Comment:
'0' values are replaced by - in the htmlpopup
p.e.
the following [identify](https://mf-chsdi3.dev.bgdi.ch/ltgud/rest/services/all/MapServer/identify?geometry=584758.75,243106.25&geometryFormat=geojson&geometryType=esriGeometryPoint&imageDisplay=837,813,96&lang=de&layers=all:ch.bafu.gewaesserschutz-diffuse_eintraege_phosphor&mapExtent=583651.25,241868.75,585743.75,243901.25&returnGeometry=true&tolerance=10) results in this [htmlpopup](https://mf-chsdi3.dev.bgdi.ch/ltgud/rest/services/ech/MapServer/ch.bafu.gewaesserschutz-diffuse_eintraege_phosphor/993_528/htmlPopup?coord=584758.75,243106.25&imageDisplay=837,813,96&lang=de&mapExtent=583651.25,241868.75,585743.75,243901.25) 

i think we have to show 0 instead of -